control-plane/src/modules/auth.ts

```diff
@@ -15,6 +15,7 @@ export const jobOwnerHash = async (authHeader: string) => {
     .select({
       organizationId: data.clusters.organization_id,
       clusterId: data.clusters.id,
+      cloudEnabled: data.clusters.cloud_enabled,
```

Comment:
Temporary way that this can be gated until we have something more robust.
Needs to be set manually to gate the feature.